packages/main/src/Menu.hbs

```diff
@@ -23,9 +23,9 @@
 		<div
 			class="ui5-menu-dialog-title"
 		>
-			<div>
+			<h1>
```

Comment:
The `ui5-shellbar` is AFAIR the only UI5 component that can render `<h1>` (apart from the title which can render any H tag).

Why did you change this here? Won't this create a wrong hierarchy, considering the nesting of menus?